core/data/pack/README.MD

```diff
@@ -1,63 +1,45 @@
-# Proto-Pack format Version 1.1
+# Proto-Pack format Version 2.0
 
-## File
+## Header
 
- name   | type               | description
-------- | ------------------ | ------------
- magic  | `byte[9]`          | `'p'`, `'r'`, `'o'`, `'t'`, `'o'`, `'p'`, `'a'`, `'c'`, `'k'`
- header | `Chunk<Header>`    | Header chunk
- data   | `Chunk<Section>[]` | File content
+ name   | type       | description
+------- | ---------- | ------------
+ magic  | `byte[16]` | `"ProtoPack\r\n2.0\n\0"`
 
-## Chunk\<T\>
+The header contains both types of new-lines, which is common in file
+headers to detect corruption caused by automatic new-line conversions.
+The header is followed by arbitrary number of variable-sized chunks.
+Chunks can be either object instance or type definition depending on the
+sign of the `size` field (encoded as protobuf's variable-length zigzag).
 
- name  | type             | description
------- | ---------------- | ------------
- size  | `varint`         | Size of the chunk data
- data  | `byte[size]`     | `T` data
+## Object instance (size>0)
 
-## Section
+ name     | type      | description
+--------- | --------- | ------------
+ `size`   | `sint32`  | Total size of the chunk excluding this size field.
```

Comment:
You've killed the definition of a chunk. Do you mean object?

Reply:
Chunk is still defined in the README:
"The header is followed by arbitrary number of variable-sized chunks."
I will make it a bit more clear.